index.js

```diff
@@ -23,10 +25,21 @@ function Babel(inputTree, options) {
     return new Babel(inputTree, options);
   }
 
-  this.inputTree = inputTree;
   this.options = options || {};
   this.extensions = this.options.filterExtensions || ['js'];
   this.extensionsRegex = getExtensionsRegex(this.extensions);
+
+  if ( this.options.browserPolyfill ) {
```

Comment:
this isn't really consistent with the other code style (the spacing inside params)